src/isar/state_machine/states/cancel.py

```diff
@@ -43,7 +52,10 @@ def start(self):
                         + f"Inspection reference: {inspection}"
                     )
 
-            self.storage_service.store_metadata(self.state_machine.current_mission)
+            if inspections:
```

Comment:
Should be moved one identation to the left

Reply:
Good catch!